github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/events/impl/GitHubPRDescriptionEvent/help.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    Skip triggering when matched comment exist. To unblock delete comment.
+    If given, don't build a pull request if its description contains the given string.
```

Comment:
Need to be checked with java classes. I remember that one of events worked only with first comment of PR(Issues) that is description of issue/pr. So i named it as `description`.